.gitignore

```diff
@@ -1,2 +1,2 @@
-# Ignore output of scraper
-data.sqlite
+*.pyc
+delete_*.json
```

Comment:
The .pyc files are compiled python files and are created any time a .py file is ran. The delete_*.json files shouldn't be there as we're now storing the delete docs on s3.